backend/service-import/src/jvmMain/kotlin/dev/johnoreilly/confetti/backend/import/DevFestNantes.kt

```diff
@@ -20,7 +20,6 @@ import kotlin.time.Duration.Companion.minutes
 private val timeZone = "Europe/Paris"
 
 private val okHttpClient = OkHttpClient.Builder()
-    .fastFallback(true)
```

Comment:
Why not keeping them?

Reply:
It's defaulted on in 5.0.0-alpha.11

https://github.com/square/okhttp/blob/master/CHANGELOG.md#version-500-alpha11

Reply:
Nice!